x-pack/plugin/core/src/main/resources/logs-dlq-mappings.json

```diff
@@ -0,0 +1,33 @@
+{
+  "template": {
+    "mappings": {
+      "dynamic": "false",
+      "properties": {
+        "@timestamp": {
+          "type": "date",
+          "ignore_malformed": false
+        },
+        "data_stream": {
+          "properties": {
+            "type": {
+              "type": "constant_keyword",
+              "value": "logs"
+            },
+            "dataset": {
+              "type": "constant_keyword",
+              "value": "dlq"
+            },
+            "namespace": {
+              "type": "constant_keyword"
```

Comment:
Does the dlq data stream need a namespace?

Reply:
Yes, I think every data stream should have a namespace. The question is whether it should be customizable or be fixed to `default`. I think a dlq per namespace makes sense. It helps with managing ILM and permissions without the dlqs being too fine-grained and therefore hard to manage, such as having one per data stream.